### DISCUSSION TITLE
Can I have my repository hosted in Australia

### DISCUSSION THREAD

**Discussion by Kamiho13** [community/community#23090] [Enterprise]
<p>For my software development I need the repository to be hosted in Australia, is it possible with the repositories to select where my repository is hosted?</p>

**Comment by jfullstackdev** [comment]
<p>you mean the data centers ? what benefit would that be ? when GitHub is not that much demanding when it comes to data and loading</p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/kamiho13">@Kamiho13</a> you cannot choose where your repository is hosted.  it will be hosted in US data centres.</p>
<p>The only GitHub managed Cloud offering that offers data residency will be GitHub AE but that is still an emerging product, intended for larger enterprises with a data residency requirement (perhaps for legal reasons) and expect significant additional costs.</p>

**Comment by mpboom** [comment]
<p>Having “hosting in AUS” as a requirement seems a bit odd to me. It’s just code. If your code is so secret you have requirements around hosting, hosted GitHub probably isn’t right for you. Self-host something on an environment you can control (a <a href="https://about.gitlab.com/install/" rel="noopener nofollow ugc">GitLab</a> instance on a server close to you, perhaps?).</p>
<p>If you represent a large(r) company, you may check with Sales if an on-premise version of GitHub is available:<br>
<a href="https://github.com/enterprise/contact" rel="noopener nofollow ugc">https://github.com/enterprise/contact</a></p>

**Comment by Kamiho13** [comment]
<p>Thank-you for your response, it was very helpful.</p>

**Comment by Kamiho13** [comment]
<p>Thank-you for your response, a client has the requirement to host in AU. So until AE is fully rolled out we will host ourselves.</p>

**Comment by Kamiho13** [comment]
<p>Thank-you for your response, a client has the requirement to host in AU. Will wait until AE is fully rolled out.</p>

**Comment by TarunMathew** [comment]
Is this option available if we are using enterprise GH? I see in the organisation settings page an option to select Location or is there any other way to chose where the code is stored without using GitHubAE when using enterprise GH?
